app/src/main/java/com/example/streammoviesapplication/data/db/TrendingMoviesDao.kt

```diff
@@ -0,0 +1,20 @@
+package com.example.streammoviesapplication.data.db
+
+import androidx.room.Dao
+import androidx.room.Insert
+import androidx.room.OnConflictStrategy
+import androidx.room.Query
+import com.example.streammoviesapplication.data.trendingMovies.localData.TrendingMoviesEntity
+import com.example.streammoviesapplication.utils.Constants.MOVIE_DATABASE
+
+@Dao
+interface TrendingMoviesDao {
+
+    @Insert(onConflict = OnConflictStrategy.REPLACE)
+    suspend fun insertTrendingMoviesToDatabase(movies: List<TrendingMoviesEntity>)
```

Comment:
rename this to `insertTrendingMovies`. we already know dao is attached to database

Reply:
done. I thought we were going to have only one dao thats why i used the previous naming conventions.